library/kani/src/arbitrary.rs

```diff
@@ -118,3 +118,26 @@ where
         if bool::any() { Ok(T::any()) } else { Err(E::any()) }
     }
 }
+
+macro_rules! tuple_arbitrary {
+    ($($typ:ident),*) => {
+        impl<$($typ : Arbitrary),*> Arbitrary for ($($typ,)*) {
+            fn any() -> Self {
+                ($($typ::any(),)*)
+            }
+        }
+    }
+}
+
+tuple_arbitrary!(A);
```

Comment:
what is this doing?

Reply:
Implements kani::Arbitrary for Tuples where each element already implements kani::Arbitrary.

This is needed b/c proptests generate the types as tuples so that their macros work well.